### PR TITLE
6 create comments

### LINKED ISSUE #6: Create Comment
`POST /api/v1/contributions/:id/comments`
- [x] Takes in the following JSON shape

```javascript
{
  user_id: 123
  body: "This is a kind comment on someone's cool contribution"
}
``` 
- [x] Returns a 201 if successful
- [x] Returns a 400 if not successful

### PR DESCRIPTION
**Pull request by jude-lawson**
@jude-lawson Wrote Tests
@jude-lawson Implemented 
@zdcthomas  Reviewed

# Neccesary checkmarks:
- [X] All Tests are Passing
- [ ] The code will run locally

## Type of change
- [X] New feature 
- [ ] Bug Fix

# Implements/Fixes:
- Implements Comments#create action for creating comments as part of contributions
closes #6 

# Check the correct boxes
- [X] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything

# Testing Changes
- [X] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened)

# Checklist:
- [X] My code has no unused/commented out code
- [X] I have reviewed my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have fully tested my code

# Notes:
- 

# Code Snippets of Changes:
```

```

(╯°□°）╯︵ ┻━┻

### CODE REVIEW

File: app/controllers/api/v1/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Api::V1::CommentsController < ApplicationController
+  def create
+    begin
```
Comment (zdcthomas) [inline app/controllers/api/v1/comments_controller.rb]: I swear i'm gonna refactor this form

**Comment by zdcthomas** [comment]
Go ahead an rebase this and I'll merge it up

**Comment by jude-lawson** [comment]
All rebased. @zdcthomas please ignore the CircleCI build failing for now. Its configuration is in progress.

**Comment by RyanH5** [comment]
Nice.

**Pull request merged**